### ISSUE TITLE
Feature to save and display the highscore

### ISSUE THREAD

**Issue by kshavp** [awaismirza/Java-Game-Brick-Breaker#14]
Local highscore can be saved and then displayed on the next gameplay.

@awaismirza assign this issue to me.

**Comment by aryadavare19** [comment]
yes I can store the Local High Score and then display it for new game target , please assign it to me.

**Comment by Swarali-31** [comment]
I would like to work on this. @awaismirza

**Comment by himeshkumar2007** [comment]
Excited to contribute to GSSC24! As a Java developer with expertise in Swing and JDBC, proposing to implement a high score feature for the Java Brick Breaker game. Will ensure efficient database management and robust authentication. Open to feedback. Thanks!
@awaismirza

**Comment by AtulYadavCodes** [comment]
i would like to work on this under gssoc 24 . i will also submit this by today itself
thanks

**Comment by AnushkaSharma04** [comment]
@awaismirza I am a gssoc'24 contributor and have experience in java and jdbc . Kindly assign this issue to me

**Comment by Jeet-0510** [comment]
Hello @awaismirza sir. I have Experience in Java from over 1 year and I have built many projects in java. I would like to start my first open-source contribution with your project. Please Assign this issue to me. I will give my Best for this.

**Comment by grasyPatel** [comment]
Hello , i am gssoc'24 contributor , I would like to work on this. Kindly assign this issue to me
